GoogleForCreators/web-stories-wp
# Dashboard: Fix template insertion

## Context



#12237 accidentally broke the "Use template" functionality on the dashboard as it turns out the element types were in fact used after all. It wasn't caught because of a missing (disabled) e2e test.

But now no more!

## Summary



Instead of just reverting #12237, this PR removes usage of the `output` package from the dashboard to fix the template insertion functionality.

## Relevant Technical Choices



Fixes and re-enables the relevant e2e test.

## To-do



## User-facing changes



No visual changes, just a fixed user flow

## Testing Instructions






This PR can be tested by following these steps:

1. Go to "Explore Templates"
2. Create a new story from a template
3. It should succeed


## Reviews

### Does this PR have a security-related impact?



No

### Does this PR change what data or activity we track or use?



No

### Does this PR have a legal-related impact?



No

## Checklist




---



Fixes #12345 
Fixes #12024
